### PR TITLE
added datasets-sars-cov-2

### PR DESCRIPTION
**Pull request by k-florek**
Added docker container for getting the QC datasets via [https://github.com/CDCgov/datasets-sars-cov-2/tree/master](https://github.com/CDCgov/datasets-sars-cov-2/tree/master)

### CODE REVIEW

**Comment by erinyoung** [comment]
Do you still need to set an email with this? 

Is the usage any different?

**Comment by k-florek** [comment]
The usage shouldn't be any different. You don't need an email, only a NCBI_API_KEY. I'll probably open a PR on the CDC github repo with instructions on how to use it.

**Comment by kapsakcj** [comment]
Sorry for the delay, I tried this image out a few weeks ago and got stuck. Think I've figured it out though.

Image builds successfully (with Kelsey's most recent commits to update to v0.6.2). It does take quite some time to build maybe 10-15 minutes, given all the compilation of the NCBI code.

```
# spawn a container
$ docker run --rm=True -v $PWD:/data -u $(id -u):$(id -g) -ti kflorek/sc2datasets:0.6.2

# set API key as env variable
$ export NCBI_API_KEY="<not-my-actual-API-key>"

# download a dataset, given input TSV
$ GenFSGopher.pl -o vocvoi-output /home/user/datasets-sars-cov-2/datasets/sars-cov-2-voivoc.tsv --numcpus 8 --layout onedir --compressed

# output directory looks as expected
$ ls -lh vocvoi-output/
total 661M
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 ERR5181310
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 ERR5181360
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 ERR5405022
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 ERR5556343
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 ERR5691945
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 ERR5743043
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 ERR5743893
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 ERR5749609
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 ERR5771988
-rw-r--r-- 1 1001 1002  20K Jan 27 04:34 Makefile
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 SRR13422799
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 SRR13422802
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 SRR13500958
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 SRR13511054
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 SRR13603869
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 SRR13606410
drwxr-xr-x 2 1001 1002 4.0K Jan 27 04:34 SRR14241199
-rw-r--r-- 1 1001 1002 4.9M Jan 27 04:35 hCoV-19_England_CAMC-13E7537_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_CAMC-13E7537_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002 5.7M Jan 27 04:35 hCoV-19_England_CAMC-13E7537_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_CAMC-13E7537_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002 4.3M Jan 27 04:35 hCoV-19_England_CAMC-14E726C_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_CAMC-14E726C_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002 5.3M Jan 27 04:35 hCoV-19_England_CAMC-14E726C_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_CAMC-14E726C_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002  43M Jan 27 04:35 hCoV-19_England_CAMC-14E7A0A_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_CAMC-14E7A0A_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002  47M Jan 27 04:35 hCoV-19_England_CAMC-14E7A0A_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_CAMC-14E7A0A_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002  28M Jan 27 04:35 hCoV-19_England_CAMC-1510779_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_CAMC-1510779_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002  29M Jan 27 04:35 hCoV-19_England_CAMC-1510779_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_CAMC-1510779_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002  43M Jan 27 04:35 hCoV-19_England_QEUH-14A3615_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_QEUH-14A3615_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002  45M Jan 27 04:35 hCoV-19_England_QEUH-14A3615_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_QEUH-14A3615_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002  20M Jan 27 04:35 hCoV-19_England_QEUH-E93AAC_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  107 Jan 27 04:35 hCoV-19_England_QEUH-E93AAC_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002  21M Jan 27 04:35 hCoV-19_England_QEUH-E93AAC_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  107 Jan 27 04:35 hCoV-19_England_QEUH-E93AAC_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002  33M Jan 27 04:35 hCoV-19_England_QEUH-E948CF_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  107 Jan 27 04:35 hCoV-19_England_QEUH-E948CF_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002  36M Jan 27 04:35 hCoV-19_England_QEUH-E948CF_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  107 Jan 27 04:35 hCoV-19_England_QEUH-E948CF_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002  58M Jan 27 04:35 hCoV-19_England_RAND-14DD366_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_RAND-14DD366_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002  58M Jan 27 04:35 hCoV-19_England_RAND-14DD366_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  108 Jan 27 04:35 hCoV-19_England_RAND-14DD366_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002  15M Jan 27 04:35 hCoV-19_USA_CA-QDX-2761_2020_1.fastq.gz
-rw-r--r-- 1 1001 1002  103 Jan 27 04:35 hCoV-19_USA_CA-QDX-2761_2020_1.fastq.sha256
-rw-r--r-- 1 1001 1002  23M Jan 27 04:35 hCoV-19_USA_CA-QDX-2761_2020_2.fastq.gz
-rw-r--r-- 1 1001 1002  103 Jan 27 04:35 hCoV-19_USA_CA-QDX-2761_2020_2.fastq.sha256
-rw-r--r-- 1 1001 1002 7.3M Jan 27 04:35 hCoV-19_USA_CA-QDX-2781_2020_1.fastq.gz
-rw-r--r-- 1 1001 1002  103 Jan 27 04:35 hCoV-19_USA_CA-QDX-2781_2020_1.fastq.sha256
-rw-r--r-- 1 1001 1002  13M Jan 27 04:35 hCoV-19_USA_CA-QDX-2781_2020_2.fastq.gz
-rw-r--r-- 1 1001 1002  103 Jan 27 04:35 hCoV-19_USA_CA-QDX-2781_2020_2.fastq.sha256
-rw-r--r-- 1 1001 1002 6.9M Jan 27 04:35 hCoV-19_USA_MA-QDX-2966_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  103 Jan 27 04:35 hCoV-19_USA_MA-QDX-2966_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002 8.3M Jan 27 04:35 hCoV-19_USA_MA-QDX-2966_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  103 Jan 27 04:35 hCoV-19_USA_MA-QDX-2966_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002  11M Jan 27 04:35 hCoV-19_USA_NV-QDX-4024_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  103 Jan 27 04:35 hCoV-19_USA_NV-QDX-4024_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002  12M Jan 27 04:35 hCoV-19_USA_NV-QDX-4024_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  103 Jan 27 04:35 hCoV-19_USA_NV-QDX-4024_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002  11M Jan 27 04:35 hCoV-19_USA_RI-QDX-3232_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  103 Jan 27 04:35 hCoV-19_USA_RI-QDX-3232_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002  15M Jan 27 04:35 hCoV-19_USA_RI-QDX-3232_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  103 Jan 27 04:35 hCoV-19_USA_RI-QDX-3232_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002 2.8M Jan 27 04:35 hCoV-19_USA_TX-DSHS-2263_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  104 Jan 27 04:35 hCoV-19_USA_TX-DSHS-2263_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002 3.0M Jan 27 04:35 hCoV-19_USA_TX-DSHS-2263_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  104 Jan 27 04:35 hCoV-19_USA_TX-DSHS-2263_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002  24M Jan 27 04:35 hCoV-19_USA_TX-UTA-000313_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  105 Jan 27 04:35 hCoV-19_USA_TX-UTA-000313_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002  27M Jan 27 04:35 hCoV-19_USA_TX-UTA-000313_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  105 Jan 27 04:35 hCoV-19_USA_TX-UTA-000313_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002 4.8M Jan 27 04:35 hCoV-19_Wales_PHWC-4C8F5E_2021_1.fastq.gz
-rw-r--r-- 1 1001 1002  105 Jan 27 04:35 hCoV-19_Wales_PHWC-4C8F5E_2021_1.fastq.sha256
-rw-r--r-- 1 1001 1002 5.1M Jan 27 04:35 hCoV-19_Wales_PHWC-4C8F5E_2021_2.fastq.gz
-rw-r--r-- 1 1001 1002  105 Jan 27 04:35 hCoV-19_Wales_PHWC-4C8F5E_2021_2.fastq.sha256
-rw-r--r-- 1 1001 1002 4.0K Jan 27 04:34 in.tsv
-rw-r--r-- 1 1001 1002    0 Jan 27 04:34 prefetch.done
```
When I ran the above command but without `--compressed` option, all md5sums check out OK ✔️ 

With the `--compressed` option added, md5sums check out OK ✔️  and it prints the file size reduction when compressing the fastq files.
```
hCoV-19_England_CAMC-13E7537_2021_1.fastq:      hCoV-19_England_CAMC-13E7537_2021_1.fastq: OK
hCoV-19_England_CAMC-13E7537_2021_2.fastq: OK
hCoV-19_England_CAMC-14E726C_2021_1.fastq: OK
hCoV-19_England_CAMC-14E726C_2021_2.fastq: OK
 94.9% -- replaced with hCoV-19_England_CAMC-13E7537_2021_1.fastq.gz
hCoV-19_England_CAMC-13E7537_2021_2.fastq:       93.5% -- replaced with hCoV-19_England_CAMC-13E7537_2021_2.fastq.gz
hCoV-19_England_CAMC-14E726C_2021_1.fastq:       94.3% -- replaced with hCoV-19_England_CAMC-14E726C_2021_1.fastq.gz
hCoV-19_England_CAMC-14E726C_2021_2.fastq:      hCoV-19_England_CAMC-14E7A0A_2021_1.fastq: OK
 92.2% -- replaced with hCoV-19_England_CAMC-14E726C_2021_2.fastq.gz
hCoV-19_England_CAMC-14E7A0A_2021_1.fastq:      hCoV-19_England_CAMC-14E7A0A_2021_2.fastq: OK
hCoV-19_England_CAMC-1510779_2021_1.fastq: OK
hCoV-19_England_CAMC-1510779_2021_2.fastq: OK
hCoV-19_England_QEUH-14A3615_2021_1.fastq: OK
hCoV-19_England_QEUH-14A3615_2021_2.fastq: OK
 94.9% -- replaced with hCoV-19_England_CAMC-14E7A0A_2021_1.fastq.gz
hCoV-19_England_CAMC-14E7A0A_2021_2.fastq:      hCoV-19_England_QEUH-E93AAC_2021_1.fastq: OK
hCoV-19_England_QEUH-E93AAC_2021_2.fastq: OK
hCoV-19_England_QEUH-E948CF_2021_1.fastq: OK
hCoV-19_England_QEUH-E948CF_2021_2.fastq: OK
hCoV-19_England_RAND-14DD366_2021_1.fastq: OK
 93.9% -- replaced with hCoV-19_England_CAMC-14E7A0A_2021_2.fastq.gz
hCoV-19_England_CAMC-1510779_2021_1.fastq:      hCoV-19_England_RAND-14DD366_2021_2.fastq: OK
hCoV-19_USA_CA-QDX-2761_2020_1.fastq: OK
hCoV-19_USA_CA-QDX-2761_2020_2.fastq: OK
hCoV-19_USA_CA-QDX-2781_2020_1.fastq: OK
hCoV-19_USA_CA-QDX-2781_2020_2.fastq: OK
hCoV-19_USA_MA-QDX-2966_2021_1.fastq: OK
hCoV-19_USA_MA-QDX-2966_2021_2.fastq: OK
hCoV-19_USA_NV-QDX-4024_2021_1.fastq: OK
hCoV-19_USA_NV-QDX-4024_2021_2.fastq: OK
hCoV-19_USA_RI-QDX-3232_2021_1.fastq: OK
hCoV-19_USA_RI-QDX-3232_2021_2.fastq: OK
hCoV-19_USA_TX-DSHS-2263_2021_1.fastq: OK
hCoV-19_USA_TX-DSHS-2263_2021_2.fastq: OK
hCoV-19_USA_TX-UTA-000313_2021_1.fastq: OK
hCoV-19_USA_TX-UTA-000313_2021_2.fastq: OK
hCoV-19_Wales_PHWC-4C8F5E_2021_1.fastq: OK
hCoV-19_Wales_PHWC-4C8F5E_2021_2.fastq: OK
 94.4% -- replaced with hCoV-19_England_CAMC-1510779_2021_1.fastq.gz
hCoV-19_England_CAMC-1510779_2021_2.fastq:       93.7% -- replaced with hCoV-19_England_CAMC-1510779_2021_2.fastq.gz
hCoV-19_England_QEUH-14A3615_2021_1.fastq:       94.5% -- replaced with hCoV-19_England_QEUH-14A3615_2021_1.fastq.gz
hCoV-19_England_QEUH-14A3615_2021_2.fastq:       93.7% -- replaced with hCoV-19_England_QEUH-14A3615_2021_2.fastq.gz
hCoV-19_England_QEUH-E93AAC_2021_1.fastq:        95.1% -- replaced with hCoV-19_England_QEUH-E93AAC_2021_1.fastq.gz
hCoV-19_England_QEUH-E93AAC_2021_2.fastq:        94.5% -- replaced with hCoV-19_England_QEUH-E93AAC_2021_2.fastq.gz
hCoV-19_England_QEUH-E948CF_2021_1.fastq:        95.1% -- replaced with hCoV-19_England_QEUH-E948CF_2021_1.fastq.gz
hCoV-19_England_QEUH-E948CF_2021_2.fastq:        94.1% -- replaced with hCoV-19_England_QEUH-E948CF_2021_2.fastq.gz
hCoV-19_England_RAND-14DD366_2021_1.fastq:       94.5% -- replaced with hCoV-19_England_RAND-14DD366_2021_1.fastq.gz
hCoV-19_England_RAND-14DD366_2021_2.fastq:       93.9% -- replaced with hCoV-19_England_RAND-14DD366_2021_2.fastq.gz
hCoV-19_USA_CA-QDX-2761_2020_1.fastq:    80.5% -- replaced with hCoV-19_USA_CA-QDX-2761_2020_1.fastq.gz
hCoV-19_USA_CA-QDX-2761_2020_2.fastq:    68.4% -- replaced with hCoV-19_USA_CA-QDX-2761_2020_2.fastq.gz
hCoV-19_USA_CA-QDX-2781_2020_1.fastq:    80.6% -- replaced with hCoV-19_USA_CA-QDX-2781_2020_1.fastq.gz
hCoV-19_USA_CA-QDX-2781_2020_2.fastq:    67.7% -- replaced with hCoV-19_USA_CA-QDX-2781_2020_2.fastq.gz
hCoV-19_USA_MA-QDX-2966_2021_1.fastq:    80.8% -- replaced with hCoV-19_USA_MA-QDX-2966_2021_1.fastq.gz
hCoV-19_USA_MA-QDX-2966_2021_2.fastq:    76.9% -- replaced with hCoV-19_USA_MA-QDX-2966_2021_2.fastq.gz
hCoV-19_USA_NV-QDX-4024_2021_1.fastq:    77.8% -- replaced with hCoV-19_USA_NV-QDX-4024_2021_1.fastq.gz
hCoV-19_USA_NV-QDX-4024_2021_2.fastq:    73.7% -- replaced with hCoV-19_USA_NV-QDX-4024_2021_2.fastq.gz
hCoV-19_USA_RI-QDX-3232_2021_1.fastq:    78.9% -- replaced with hCoV-19_USA_RI-QDX-3232_2021_1.fastq.gz
hCoV-19_USA_RI-QDX-3232_2021_2.fastq:    71.9% -- replaced with hCoV-19_USA_RI-QDX-3232_2021_2.fastq.gz
hCoV-19_USA_TX-DSHS-2263_2021_1.fastq:   74.3% -- replaced with hCoV-19_USA_TX-DSHS-2263_2021_1.fastq.gz
hCoV-19_USA_TX-DSHS-2263_2021_2.fastq:   73.0% -- replaced with hCoV-19_USA_TX-DSHS-2263_2021_2.fastq.gz
hCoV-19_USA_TX-UTA-000313_2021_1.fastq:  79.2% -- replaced with hCoV-19_USA_TX-UTA-000313_2021_1.fastq.gz
hCoV-19_USA_TX-UTA-000313_2021_2.fastq:  76.2% -- replaced with hCoV-19_USA_TX-UTA-000313_2021_2.fastq.gz
hCoV-19_Wales_PHWC-4C8F5E_2021_1.fastq:  89.2% -- replaced with hCoV-19_Wales_PHWC-4C8F5E_2021_1.fastq.gz
hCoV-19_Wales_PHWC-4C8F5E_2021_2.fastq:  88.6% -- replaced with hCoV-19_Wales_PHWC-4C8F5E_2021_2.fastq.gz
Done compressing. Note that this removes uncompressed targets.
```

Just have a few in-line suggestions that I'll make, then I'm good to merge the PR

**Review by kapsakcj — CHANGES_REQUESTED** [review]
This docker image looks great and tested fine with the voi-voc dataset from the CDC repo. I approve the PR (with some minor edits)

I want to make a general recommendation, not necessarily to this dockerfile (too much work at this point), but others in the future.

When you are forced to update things like gcc/g++, java/openJDK, cmake, etc. other lower-level packages, I would recommending going with a more recent base image, specifically `ubuntu:focal`. Oftentimes these more recent distros will have the updated versions of compilers/java/etc and won't require manual installation. For example, on `ubuntu:focal`, the default `gcc` version in `apt` is 9.3.0 https://packages.ubuntu.com/focal/gcc <-- The search functionality on this site is horrible, but it helps you figure out which versions are available on a given ubuntu distro

This would help to simplify the dockerfile, lessen the size of the final docker image, and save you some time.

File: datasets-sars-cov-2/0.6.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+LABEL software.version="v0.6.2"
+LABEL dockerfile.version="1"
+LABEL description="Benchmark datasets for WGS analysis of SARS-CoV-2."
+LABEL website="https://github.com/CDCgov/datasets-sars-cov-2"
+LABEL license="https://github.com/CDCgov/datasets-sars-cov-2/blob/master/LICENSE"
+LABEL maintainer="Kelsey Florek"
+LABEL maintainer.email="kelsey.florek@slh.wisc.edu"
+
+# version to build with
+ARG CANU_VERSION=2.2
```
[the remark this answers is not in the document]
Comment (k-florek) [reply datasets-sars-cov-2/0.6.2/Dockerfile:16]: good catch!

File: datasets-sars-cov-2/0.6.2/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ libmagic-dev\
+ libhdf5-dev\
+ git\
+ openjdk-8-jdk\
+ software-properties-common\
+ lsb-release\
+ apt-transport-https\
+ ca-certificates\
+ doxygen\
+ uuid-runtime
```
[the remark this answers is not in the document]
Comment (k-florek) [reply datasets-sars-cov-2/0.6.2/Dockerfile]: good idea!

**Comment by k-florek** [comment]
Updated the Dockerfile with your suggestions. I also upgraded to focal since that makes a lot more sense!

**Comment by kapsakcj** [comment]
Awesome! thanks for making the adjustments. I'll give it a go

**Comment by kapsakcj** [comment]
@k-florek Do you know the min required version of `cmake` ? If we install via `apt` it would likely reduce the image size further.

`cmake` in `apt` for `ubuntu:focal` is v3.16.3

whereas the `cmake` installed via the method in the dockerfile is v3.22.2

**Comment by kapsakcj** [comment]
Either way the container ran as expected ✔️

Here's the output from hash checking and compression done at the end. I'm guessing the STDOUT format is all wonky since these processes are run in parallel.
```
hCoV-19_England_CAMC-13E7537_2021_1.fastq:      hCoV-19_England_CAMC-13E7537_2021_1.fastq: OK
hCoV-19_England_CAMC-13E7537_2021_2.fastq: OK
 94.9% -- replaced with hCoV-19_England_CAMC-13E7537_2021_1.fastq.gz
hCoV-19_England_CAMC-13E7537_2021_2.fastq:      hCoV-19_England_CAMC-14E726C_2021_1.fastq: OK
hCoV-19_England_CAMC-14E726C_2021_2.fastq: OK
 93.5% -- replaced with hCoV-19_England_CAMC-13E7537_2021_2.fastq.gz
hCoV-19_England_CAMC-14E726C_2021_1.fastq:       94.3% -- replaced with hCoV-19_England_CAMC-14E726C_2021_1.fastq.gz
hCoV-19_England_CAMC-14E726C_2021_2.fastq:       92.2% -- replaced with hCoV-19_England_CAMC-14E726C_2021_2.fastq.gz
hCoV-19_England_CAMC-14E7A0A_2021_1.fastq:      hCoV-19_England_CAMC-14E7A0A_2021_1.fastq: OK
hCoV-19_England_CAMC-14E7A0A_2021_2.fastq: OK
hCoV-19_England_CAMC-1510779_2021_1.fastq: OK
hCoV-19_England_CAMC-1510779_2021_2.fastq: OK
hCoV-19_England_QEUH-14A3615_2021_1.fastq: OK
 94.9% -- replaced with hCoV-19_England_CAMC-14E7A0A_2021_1.fastq.gz
hCoV-19_England_CAMC-14E7A0A_2021_2.fastq:      hCoV-19_England_QEUH-14A3615_2021_2.fastq: OK
hCoV-19_England_QEUH-E93AAC_2021_1.fastq: OK
hCoV-19_England_QEUH-E93AAC_2021_2.fastq: OK
hCoV-19_England_QEUH-E948CF_2021_1.fastq: OK
hCoV-19_England_QEUH-E948CF_2021_2.fastq: OK
 93.9% -- replaced with hCoV-19_England_CAMC-14E7A0A_2021_2.fastq.gz
hCoV-19_England_CAMC-1510779_2021_1.fastq:      hCoV-19_England_RAND-14DD366_2021_1.fastq: OK
hCoV-19_England_RAND-14DD366_2021_2.fastq: OK
hCoV-19_USA_CA-QDX-2761_2020_1.fastq: OK
 94.4% -- replaced with hCoV-19_England_CAMC-1510779_2021_1.fastq.gz
hCoV-19_England_CAMC-1510779_2021_2.fastq:      hCoV-19_USA_CA-QDX-2761_2020_2.fastq: OK
hCoV-19_USA_CA-QDX-2781_2020_1.fastq: OK
hCoV-19_USA_CA-QDX-2781_2020_2.fastq: OK
hCoV-19_USA_MA-QDX-2966_2021_1.fastq: OK
hCoV-19_USA_MA-QDX-2966_2021_2.fastq: OK
hCoV-19_USA_NV-QDX-4024_2021_1.fastq: OK
hCoV-19_USA_NV-QDX-4024_2021_2.fastq: OK
hCoV-19_USA_RI-QDX-3232_2021_1.fastq: OK
hCoV-19_USA_RI-QDX-3232_2021_2.fastq: OK
hCoV-19_USA_TX-DSHS-2263_2021_1.fastq: OK
hCoV-19_USA_TX-DSHS-2263_2021_2.fastq: OK
hCoV-19_USA_TX-UTA-000313_2021_1.fastq: OK
hCoV-19_USA_TX-UTA-000313_2021_2.fastq: OK
hCoV-19_Wales_PHWC-4C8F5E_2021_1.fastq: OK
hCoV-19_Wales_PHWC-4C8F5E_2021_2.fastq: OK
 93.7% -- replaced with hCoV-19_England_CAMC-1510779_2021_2.fastq.gz
hCoV-19_England_QEUH-14A3615_2021_1.fastq:       94.5% -- replaced with hCoV-19_England_QEUH-14A3615_2021_1.fastq.gz
hCoV-19_England_QEUH-14A3615_2021_2.fastq:       93.7% -- replaced with hCoV-19_England_QEUH-14A3615_2021_2.fastq.gz
hCoV-19_England_QEUH-E93AAC_2021_1.fastq:        95.1% -- replaced with hCoV-19_England_QEUH-E93AAC_2021_1.fastq.gz
hCoV-19_England_QEUH-E93AAC_2021_2.fastq:        94.5% -- replaced with hCoV-19_England_QEUH-E93AAC_2021_2.fastq.gz
hCoV-19_England_QEUH-E948CF_2021_1.fastq:        95.1% -- replaced with hCoV-19_England_QEUH-E948CF_2021_1.fastq.gz
hCoV-19_England_QEUH-E948CF_2021_2.fastq:        94.1% -- replaced with hCoV-19_England_QEUH-E948CF_2021_2.fastq.gz
hCoV-19_England_RAND-14DD366_2021_1.fastq:       94.5% -- replaced with hCoV-19_England_RAND-14DD366_2021_1.fastq.gz
hCoV-19_England_RAND-14DD366_2021_2.fastq:       93.9% -- replaced with hCoV-19_England_RAND-14DD366_2021_2.fastq.gz
hCoV-19_USA_CA-QDX-2761_2020_1.fastq:    80.5% -- replaced with hCoV-19_USA_CA-QDX-2761_2020_1.fastq.gz
hCoV-19_USA_CA-QDX-2761_2020_2.fastq:    68.4% -- replaced with hCoV-19_USA_CA-QDX-2761_2020_2.fastq.gz
hCoV-19_USA_CA-QDX-2781_2020_1.fastq:    80.6% -- replaced with hCoV-19_USA_CA-QDX-2781_2020_1.fastq.gz
hCoV-19_USA_CA-QDX-2781_2020_2.fastq:    67.7% -- replaced with hCoV-19_USA_CA-QDX-2781_2020_2.fastq.gz
hCoV-19_USA_MA-QDX-2966_2021_1.fastq:    80.8% -- replaced with hCoV-19_USA_MA-QDX-2966_2021_1.fastq.gz
hCoV-19_USA_MA-QDX-2966_2021_2.fastq:    76.9% -- replaced with hCoV-19_USA_MA-QDX-2966_2021_2.fastq.gz
hCoV-19_USA_NV-QDX-4024_2021_1.fastq:    77.8% -- replaced with hCoV-19_USA_NV-QDX-4024_2021_1.fastq.gz
hCoV-19_USA_NV-QDX-4024_2021_2.fastq:    73.7% -- replaced with hCoV-19_USA_NV-QDX-4024_2021_2.fastq.gz
hCoV-19_USA_RI-QDX-3232_2021_1.fastq:    78.9% -- replaced with hCoV-19_USA_RI-QDX-3232_2021_1.fastq.gz
hCoV-19_USA_RI-QDX-3232_2021_2.fastq:    71.9% -- replaced with hCoV-19_USA_RI-QDX-3232_2021_2.fastq.gz
hCoV-19_USA_TX-DSHS-2263_2021_1.fastq:   74.3% -- replaced with hCoV-19_USA_TX-DSHS-2263_2021_1.fastq.gz
hCoV-19_USA_TX-DSHS-2263_2021_2.fastq:   73.0% -- replaced with hCoV-19_USA_TX-DSHS-2263_2021_2.fastq.gz
hCoV-19_USA_TX-UTA-000313_2021_1.fastq:  79.2% -- replaced with hCoV-19_USA_TX-UTA-000313_2021_1.fastq.gz
hCoV-19_USA_TX-UTA-000313_2021_2.fastq:  76.2% -- replaced with hCoV-19_USA_TX-UTA-000313_2021_2.fastq.gz
hCoV-19_Wales_PHWC-4C8F5E_2021_1.fastq:  89.2% -- replaced with hCoV-19_Wales_PHWC-4C8F5E_2021_1.fastq.gz
hCoV-19_Wales_PHWC-4C8F5E_2021_2.fastq:  88.6% -- replaced with hCoV-19_Wales_PHWC-4C8F5E_2021_2.fastq.gz
Done compressing. Note that this removes uncompressed targets.
DONE! If you used this script in a publication, please cite us at github.com/WGS-standards-and-analysis/datasets
make: Leaving directory '/data/vocvoi-output-ubuntu-focal'

```

**Review by kapsakcj — APPROVED** [review]
Thanks for making the changes I suggested! I approve this PR

**Pull request merged**